### DISCUSSION TITLE
Leveraging K-12 LO framework for professional competency tracking

### DISCUSSION THREAD

**Discussion by sagarihanda** [Sunbird-Knowlg/Community#40] [General Discussions]
**Context**
UP State Medical Faculty (SMF) is building a platform to manage human resources for allied healthcare professionals (nurses, paramedical staff, lab technicians etc.) in the state. The Government is looking to build the platform on Sunbird building blocks to adhere to ABDM's (https://abdm.gov.in/abdm) principles. One of the key modules being built is to enable competency based assessments of allied professionals, and to enable mid-career learning courses via an LMS. Six Sunbird blocks (RC, Obsrv, Knowlg, inQuiry, CoKreat and Ed) are being leveraged to develop the platform with the user data being housed in Sunbird RC.

**Clarification**
The UP SMF will follow the FRAC framework (https://dopttrg.nic.in/igotmk/ImportantDocuments/Part1%20FRAC.pdf) to define functions, roles, activities and competencies for each profession under the healthcare professionals umbrella. Can the K-12 LO (learning outcome) framework be expanded to incorporate this?

@rahul101001000

**Comment by AmiableAnil** [comment]
Please refer to [this](https://github.com/Sunbird-Knowlg/Community/discussions/45)

**Reply by pallakartheekreddy** [reply]
@sagarihanda As we haven't received a response from you, we will proceed to close this thread. If you have any further questions or require assistance in the future, please don't hesitate to start a new thread. Thank you.
